README.md

```diff
@@ -100,8 +100,8 @@ module.exports = {
       '/absolute/path/to/dir',
       '**/*',
       {
-        from: '**/*',
-        globOptions: {
+        from: {
+          glob: '**/*',
```

Comment:
Keep both,

`globOptions` and `from.glob` both are supported

Reply:
Hey @anikethsaha thanks for the review!

I think just adding both options to one example makes it confusing as one of them will take precedence over the other and this example in particular is describing the `from` pattern option, not `globOptions`. Do you think it makes sense to add `globOptions` as a separate entry to the doc as all the other options on the pattern?

I also noticed that the typedefs don't support `globOptions` but if I get it right this should be fixed in [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) repo, right?

Reply:
Its actually a separate option so It should be documented as a separate options/entry.
Yes about the type it should be changed but I am not sure whether it been released or not?

cc @evilebottnawi

Reply:
Yes, globOptions and from.glob both are supported, feel free to fix it in `DefinitelyTyped`, in next major release we will remove `from.glob` in favor `globOptions` so we can rewrite documentation using `globOptions` (less misleading)

Reply:
Thanks for the feedback y'all, I'll update the PR with new section on `globOptions` then 👍